src/main/java/org/jboss/jandex/AnnotationValue.java

```diff
@@ -123,7 +123,7 @@ public static AnnotationValue createDouleValue(String name, double d) {
         return new DoubleValue(name, d);
     }
 
-    public static AnnotationValue createLongalue(String name, long l) {
```

Comment:
There is another typo in the method above `createDouleValue`.

Reply:
Good catch! Fixed. I also corrected two Javadoc typos.

I guess I should make the old methods deprecated and add the corrected signatures to make this fix non-breaking. WDYT?

Reply:
Yes, I think that would be safer.

Reply:
The old methods are back and deprecated, this is now a non-breaking PR :)